src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -91,7 +92,7 @@ static size_t get_dynamic_quantize_group_size(const fully_connected_params& para
     return 0;
 }
 
-static bool should_dynamic_quantize(const fully_connected_params& params) {
+static bool should_dynamic_quantize(const fully_connected_params& params, bool print_log = false) {
```

Comment:
I guess `print_log` is only for debugging purpose? Do we need to have it in the function argument?

Reply:
When I checked verbose=5 log, I found there were too many debug lines from this line.
Do you think we should remove this?

Reply:
That is how it is supposed to work(Logging without changing code). This is basically a dead code which is not executed unless you change the code. Could you clean up in the next PR?